### DISCUSSION TITLE
Viikon 40 päivitys

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#808] [Tiedotteet]
Viikon 40 päivitys tehdään tiistai-aamuna 3.10.2023 ja se pitää sisällään seuraavat muutokset:

## Laina- ja maksusäännöissä myöhästymismaksun veloitusaika muuttui "itsekseen"

Jos laina- ja maksusäännöissä otti jonkin sääntörivin muokkaukseen, muuttui myöhästymismaksun veloitusaika "itsekseen" toiseksi. Tämä johtui siitä, että aikavälin ja siihen liittyvän valikon käännökset erosivat toisistaan ja liittymä ei osannut täsmäyttää tietoja. Ongelma korjattiin muuttamalla käännökset vastaamaan toisiaan.

![270643065-97d3e922-e514-4179-8f60-568ad6e56ca0](https://github.com/KohaSuomi/Koha/assets/33121325/3774dab9-7168-40e9-a926-f6b171173ad1)

Liittyy tikettiin [Koha-translations #24](https://github.com/KohaSuomi/Koha-translations/issues/24).

Asiasanat: käännökset, laina- ja maksusäännöt, veloitusväli

## Asiakkaan lainaus- ja tiedot-välilehdille näkyviin kaikki asiakkaan maksut

Asiakkaan lainaus- ja tiedot-välilehdillä on tähän mennessä näkynyt vain lainauskieltoon vaikuttavien (julkisoikeudellisten, kuten myöhästymismaksu, noutamattoman varauksen maksu) maksujen summa, eikä ollenkaan käsin lisättyjä yksityisoikeudellisia maksuja (kuten esim. tulosteet, kassit, 3D-tulosteet yms.). Kaikki maksut näkyivät vasta, kun virkailija siirtyi Maksut-välilehdelle. Nyt lainaus- ja tiedot-välilehtien huomiohin on lisätty erikseen tieto myös niistä maksuista, jotka eivät vaikuta lainakieltorajaan.

![266212084-63274586-5d70-4d42-a202-2a6a063cb334](https://github.com/KohaSuomi/Koha/assets/33121325/659dc251-d793-43ee-9096-3ad8d435cf21)

Liittyy tikettiin [Koha #762](https://github.com/KohaSuomi/Koha/issues/762).

Asiasanat: asiakkaan maksut, lainakieltoraja, julkisoikeudelliset maksut, yksityisoikeudelliset maksut

## Tarkassa haussa boolen operaattoreiden valikosta pois UDK- ja YKL-valinnat

Tarkassa haussa näkyi boolen operaattoreiden valikossa virheellisesti UDK- ja YKL-luokituksen valinta. Virhe johtui liian väljästä määrityksestä JavaScript-rimpsussa, jolla lisätään tarkan haun sivulle valikoihin mm. nuo UDK- ja YKL-luokitukset sekä tekstimuutoksia välilehtien otsikoihin. Uusi korjattu rimpsu laitetaan kaikkiin kimppoihin maanantai-iltana.

![kuva](https://github.com/KohaSuomi/Koha/assets/33121325/a78f9a67-31ef-4c0b-a37c-9632e2d747e0)

Liittyy tikettiin [Koha #790](https://github.com/KohaSuomi/Koha/issues/790).

Asiasanat: tarkka haku, valikko

## Finvoice-laskutuksessa erikoismerkit katkaisivat laskutetun aineiston ajon

OUTIssa/Oulussa Finvoice-laskutuksessa erikoismerkit katkaisivat laskutetun aineiston sisäänluvun. Merkistön käsittely on nyt korjattu niin, että meille normaalit ääkköset säilyvät, mutta muissa erikoismerkeissä merkki muutetaan sen perusmuotoon.

Liittyy tikettiin [Koha #803](https://github.com/KohaSuomi/Koha/issues/803).

Asiasanat: Finvoice, laskutus, merkistöt, erikoismerkit

## TäTissä tietueen Vie-nappi vihreäksi

TäTi-tietokannassa on toiminnallisuus, jolla voi viedä tietueita Melindaan ja Vaariin. Napin oikea väri oli kadonnut versiopäivityksessä ja se näkyi harmaana tai valkoisena. Nappi on nyt muutettu vihreäksi.

![271279298-c945de47-e1b7-4035-947b-d16647906da4](https://github.com/KohaSuomi/Koha/assets/33121325/370e6e66-cc1a-451f-8c65-cae07ff714e9)

Liittyy tikettiin [koha-plugin-broadcast-biblios #1](https://github.com/KohaSuomi/koha-plugin-broadcast-biblios/issues/1).

Asiasanat: TäTi-tietokanta, Vie-nappi, napin väri

**Comment by AnneliO** [comment]
Päivityksessä oli ongelma, jonka vuoksi käännökset eivät näy vielä. Tulevat näkyviin ensi viikolla.

**Reply by AnneliO** [reply]
Samasta ongelmasta johtuen maksujen erittely lainaus- ja palautussivulla ei näy vielä. Sekin tulee näkyviin tuotantoon ensi viikon päivityksessä.

**Reply by Kati-Helle** [reply]
Tämä ei tainnut toteutua tänään tiistaina 10.10.2023 (viikon 41 päivitys)?

**Reply by emta001** [reply]
Ei toteutunut valitettavasti, sillä tuotantoja ei tällä hetkellä kyetä päivittämään. Nämä puuttuvat päivitykset tulevat siis tuotantoon heti, kun päivitys taas pelittää.

**Reply by Kati-Helle** [reply]
Kiitos Emmi tiedosta.
